src/python/aqueduct_executor/operators/connectors/tests/conf.py

```diff
@@ -21,68 +22,65 @@
 SKIP_BIGQUERY = True
 SKIP_SQLITE = True
 
-"""POSTGRES CONFIG"""
-POSTGRES_CONF = {
-    postgres._CONFIG_USERNAME_KEY: "",
-    postgres._CONFIG_PASSWORD_KEY: "",
-    postgres._CONFIG_DATABASE_KEY: "",
-    postgres._CONFIG_HOST_KEY: "",
-    # postgres._CONFIG_PORT_KEY: "",
-}
+# """POSTGRES CONFIG"""
```

Comment:
Should we uncomment these or did you find that it was easier to initialize the config object directly instead of using a dictionary?

Reply:
I commented them out because when I left them uncommented they caused an error with running the pytest command.